data/kahet/kahet missions.txt

```diff
@@ -738,3 +738,1128 @@ event "pacified ka'sei"
 	"reputation: Ka'sei" = 1
 
 event "patir mystery timer"
+
+ship "Cloak Check"
+	attributes
+		"heat dissipation" 1
+		"hull" 3
+		"mass" 100
+		"drag" 5
+		"automaton" 1
+		"thrust" 100
+		"turn" 6000
+	outfits
+		"Cloak Checker"
+	turret 0 0
+
+outfit "Cloak Checker"
+	category "Turrets"
+	thumbnail "outfit/unknown"
+	"turret mounts" -1
+	weapon
+		"velocity" 10
+		"lifetime" 1
+		"reload" 1
+		"blast radius" 20
+		"hull damage" 5
+		"no damage scaling"
+		"range override" 1000000
+		"turret turn" 360
```

Comment:
If we update the `on encounter` NPC trigger to work on your escorts in addition to your flagship, or provide a separate trigger that works on your escorts, then we could replace these with just a normal NPC with an `on encounter` node.